hypha/apply/determinations/tests/test_views.py

```diff
@@ -460,7 +461,7 @@ def get_kwargs(self, instance):
 
     def test_can_edit_determination(self):
         submission = ApplicationSubmissionFactory(status='rejected')
-        determination = DeterminationFactory(submission=submission)
+        determination = DeterminationFactory(submission=submission, author=self.user, accepted=True, submitted=True)
```

Comment:
This seems wrong, a submitted determination is not editable i think. Also why did you need to change ownership of the determination, as previously mentioned that isnt important (as long as you are staff).

Reply:
Reverted this after making the change below